htdocs/install/mysql/migration/12.0.0-13.0.0.sql

```diff
@@ -312,4 +312,4 @@ ALTER TABLE llx_actioncomm_reminder ADD INDEX idx_actioncomm_reminder_status (st
 
 ALTER TABLE llx_inventorydet ADD UNIQUE uk_inventorydet(fk_inventory, fk_warehouse, fk_product, batch);
 
-
+ALTER TABLE llx_facturedet ADD COLUMN ref_ext varchar(255) AFTER multicurrency_total_ttc;
```

Comment:
If you add a new field with migration, you must also edit the file for first installation (llx_facturedet.sql)